packages/material-ui-lab/src/SpeedDialAction/SpeedDialAction.js

```diff
@@ -107,7 +107,7 @@ class SpeedDialAction extends React.Component {
         <Button
           variant="fab"
           mini
-          className={classNames(classes.button, !open && classes.buttonClosed)}
+          className={classNames(classNameProp, classes.button, !open && classes.buttonClosed)}
```

Comment:
There is no `className` variable in the scope. I think that we can use it, no need to rename it classNameProp.